chirp/drivers/th_uv88.py

```diff
@@ -871,30 +903,53 @@ def get_settings(self):
 
         # Menu 02 - TX Channel Select
         if self._hasLCD:
-            options = ["Last Channel", "Main Channel"]
-            rx = RadioSettingValueList(
-                options, current_index=_settings.txChSelect)
-            rset = RadioSetting("basicsettings.txChSelect",
-                                "Priority Transmit", rx)
-            basic.append(rset)
+            options = ["Main Channel", "Last Channel"]
+        else:
+            options = ["Main Channel", "Last Channel",
+                       "Active Channel with Lockout Time"]
```

Comment:
Are you sure all h777-based radios without LCD support this extra setting? Or do you want to add that only for the subset of models you know have it?